client/src/components/NavBar/NavBarLinks/NavBarLinksWrapper/NavBarLinksWrapper.tsx

```diff
@@ -0,0 +1,11 @@
+import { useAuth } from '../../../../context/useAuthContext';
+import LoggedInNavBarLinks from '../loggedIn/NavBarLinks';
+import LoggedOutNavBarLinks from '../loggedOut/NavBarLinks';
+
+function NavBarLinksWrapper(): JSX.Element {
```

Comment:
```suggestion
function NavBarLinks(): JSX.Element {
```

Nitpick: This isn't a wrapper component, so I guess we can rename this 😄

Reply:
Wrapper by function but not by composition. I do see how it may be confusing. Change applied!